gentest/fixtures/YGDisplayTest.html

```diff
@@ -0,0 +1,20 @@
+<div id="display_none" style="width: 100px; height: 100px; flex-direction: row;">
+ <div style="flex-grow: 1;"></div>
+ <div style="flex-grow: 1; display:none;"></div>
+ <div style="flex-grow: 1; width: 20px;"></div>
+ <div style="flex-grow: 1; display:none; width: 20px;"></div>
+</div>
+
+
+<div id="display_none_with_child" style="width: 100px; height: 100px; flex-direction: row;">
+ <div style="flex: 1;"></div>
+ <div style="flex: 1; display:none;">
+   <div style="flex: 1; width: 20px;"></div>
+ </div>
+ <div style="flex: 1;"></div>
+</div>
+
+<div id="display_none_with_position" style="width: 100px; height: 100px; flex-direction: row;">
+ <div style="flex-grow: 1;"></div>
+ <div style="flex-grow: 1; display:none; top: 10px;"></div>
```

Comment:
If the position over hidden nodes is 0 or 10. It is 10, so we need to set the postion to 0 in ``` ZeroOutLayoutRecursivly```, not sure if this is actually a good idea, that's the reason I didn't implemented it yet. I'm curious what you think about that.